transition-scripts/values/c6af30/dev/values-gold-c6af30-dev-active.yaml

```diff
@@ -1,15 +1,36 @@
-replicaCount: 5
+replicaCount: 3
 
-service:
-  type: ClusterIP
-  port: 8443
+additionalServerOptions: >-
+  -Dkeycloak.profile.feature.impersonation=disabled
+  -Djboss.persistent.log.dir=/var/log/eap
+  -Djgroups.dns.query=sso-keycloak-ping.c6af30-dev.svc.cluster.local
+
+extraEnvs:
+  - name: QUARKUS_LOG_FILE_ROTATION_MAX_BACKUP_INDEX
+    value: "42"
+  - name: QUARKUS_LOG_FILE_ROTATION_MAX_FILE_SIZE
+    value: 1M
+  - name: QUARKUS_LOG_FILE_ROTATION_FILE_SUFFIX
+    value: .zip
+  # There are no vanity urls in c6
+  - name: KC_HOSTNAME_URL
+    value: https://sso-keycloak-c6af30-dev.apps.gold.devops.gov.bc.ca/auth/
+  - name: KC_HOSTNAME_ADMIN_URL
+    value: https://sso-keycloak-c6af30-dev.apps.gold.devops.gov.bc.ca/auth/
+
+
+resources:
```

Comment:
The memory constraint is good to be the same for all envs eh?

Reply:
Yeah they didn't need to ge modified from what was already there